recipe/conda_build_config.yaml

```
@@ -0,0 +1,3 @@
+microarch_level:
+  - 1  
+  - 3
```

these should be conditional on building for linux x86_64

```suggestion
  - 1   # [linux64]
  - 3   # [linux64]
```